src/main/java/org/terasology/additionalworlds/desertworld/DeadBushRasterizer.java

```diff
@@ -24,7 +24,8 @@ public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         for (Vector3i block : deadBushFacet.getWorldRegion()) {
             if (deadBushFacet.getWorld(block)
                 // Don't want bushes in water, now do we?
-                && !chunk.getBlock(ChunkMath.calcBlockPos(block.addY(-1))).getURI().toString().toLowerCase().equals("Core:Water")) {
+                //Surely there's a more generic way we can make sure not to place a bush on uninhabitable blocks... -Q
```

Comment:
Surely indeed, but that's more of a note for an issue, Trello card, or forum thread than a comment :-)

Much like with whitespace if you find existing comments that could be improved, turned into proper TODOs, or extracted feel free. I'd try to avoid adding in more. In this case I'd just convert to `// TODO: Better way to avoid bushes on top of unsuitable blocks, only water is checked here` or something 👍 

Conversational comments or signatures generally just add fluff that can become confusing later :-)